app/src/main/java/com/github/libretube/ui/models/CommentsViewModel.kt

```diff
@@ -56,9 +56,16 @@ class CommentsViewModel : ViewModel() {
                 Log.e(TAG(), e.toString())
                 return@launch
             }
+
             val updatedPage = commentsPage.value?.apply {
-                comments += response.comments.filterNonEmptyComments()
+                val combinedComments = this.comments + response.comments.filterNonEmptyComments()
```

Comment:
```kotlin
comments += response.comments.filterNonEmptyComments().filter { comment -> comments.none { it.commentId == comment.commented }
```

Reply:
That's more performant and doesn't impac or modify the already visible comments.

Reply:
I'll try it and see

Reply:
It works as intended, and I have pushed the commit that uses this. Thanks